### DISCUSSION TITLE
i can't run beef-xss in kali-linux

### DISCUSSION THREAD

**Discussion by SaadZhr1420** [microsoft/WSL#10687] [Q&A]
when i enter the command "beef'-xss"
it tells me this:

[i] GeoIP database is missing
[i] Run geoipupdate to download / update Maxmind GeoIP database
[*] Please wait for the BeEF service to start.
[*]
[*] You might need to refresh your browser once it opens.
[*]
[*]  Web UI: **censored**
[*]    Hook: **censored**
[*] Example: **censored**

System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down
System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down

[*] Opening Web UI (**censored**) in: 5... 4... 3... 2... 1...

**Comment by Alex9532** [comment]
Did you open the Web UI manually?
